### PR TITLE
Added information for implementators

### PR DESCRIPTION
**Pull request by GeorgeKerscher**
I took a stab at addinformation for NVDA or Jaws folks. It needs comments and discussion. It is only partially completed.

### CODE REVIEW

**Comment by gregoriopellegrino** [comment]
Hi George,
to understand how you constructed these comments: did you start from the shared Excel file where we had identified priorities?

**Comment by GeorgeKerscher** [comment]
Yes, I used the values from the spreadsheet in this repo.

**Review by clapierre — APPROVED** [review]
Looks good George.  
I noticed at least in the preview from GitHub every time you had word"xxxx" there is no space between word and your first quotation mark.

File: dpub-aria-testing/content/DPUB-ARIA-tests.xhtml [hunk 1]
```diff
@@ -38,35 +42,45 @@
...
     <h2 id="doc-credits">doc-credits Test</h2>   
     <div role="doc-credits">This text has a role of credits.</div>
-    
+<p>This is a medium priority. Many times, the word"credits" would be included in the heading associated with this item. It should pe vocalized when the verbosity is set to high.</p>
+    <p>Suggested output "credits."
     <h2 id="doc-dedication">doc-dedication Test</h2>   
     <div role="doc-dedication">This text has a role of dedication.</div>
-    
+    <p>This is a medium priority. Many times, the word"dedication" would be included in the heading associated with this item. It should pe vocalized when the verbosity is set to high.</p>
+<p>Suggested output "dedication."
```
Comment (gautierchomel) [inline dpub-aria-testing/content/DPUB-ARIA-tests.xhtml:73]: Closing P missing here.

**Review by gautierchomel — CHANGES_REQUESTED** [review]

**Comment by gautierchomel** [comment]
[Preview](https://rawcdn.githack.com/daisy/transitiontoepub/957e2d55341567ff6522f94cb4d2b93b311a79c8/dpub-aria-testing/content/DPUB-ARIA-tests.xhtml)

**Comment by GeorgeKerscher** [comment]
I put all the information from the Google Sheets repo. I also added guidance for implementors. I think it is fairly good. The actionable items may need some work.

File: dpub-aria-testing/content/DPUB-ARIA-tests.xhtml [hunk 2]
```diff
@@ -35,109 +49,158 @@
...
+<p>Suggested output "errata"</p>
     <h2 id="doc-example">doc-example Test</h2>   
     <div role="doc-example">This text has a role of example.</div>
-    
+    <p>This is a medium priority. Many times, the word"example" would be included in the heading associated with this item. It should pe vocalized when the verbosity is set to medium.</p>
+<p>Suggested output "example"</p>
     <h2 id="doc-footnote">doc-footnote Test</h2>   
     <div role="doc-footnote">This text has a role of footnote.</div>
-    
+    <p>This is a critical  priority. Many times, the word"footnote" would be included in the heading associated with this item. It should  always ve vocalized.</p>
```
Comment (clapierre) [inline dpub-aria-testing/content/DPUB-ARIA-tests.xhtml:121]: word"footnote" missing space before initial quotation mark.

**Pull request merged**